### PR TITLE
Use latest (beta) versions of cloud client libraries.

### PR DESCRIPTION
**Pull request by tswast**
Note this wasn't automatically done because dpebot currently skips beta
releases. We can look at how to add an exception for the
com.google.cloud libraries if they are going to stay beta a while.

### CODE REVIEW

File: language/cloud-client/pom.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-language</artifactId>
-      <version>0.7.0</version>
+      <version>0.8.0</version>
```
Comment (lesv) [inline language/cloud-client/pom.xml]: -beta ??

Comment (lesv) [inline language/cloud-client/pom.xml]: Sigh

**Comment by lesv** [comment]
Could you update getting-started-java ??

**Comment by tswast** [comment]
Sure. I can do that now.

**Comment by lesv** [comment]
Actually - I'll have a PR later today for that - I'll do it.

**Comment by tswast** [comment]
Okay.

**Pull request merged**